### ISSUE TITLE
How do i open the GUI?

### ISSUE THREAD

**Issue by Thimothyeet** [ffleurey/ThingML-Tradfri#4]

**Comment by Wilsas** [comment]
In the readme, under the GUI screenshot - 2nd line:

> To run the GUI, run `mvn exec:java`

**Comment by Thimothyeet** [comment]
> In the readme, under the GUI screenshot - 2nd line:
> 
> > To run the GUI, run `mvn exec:java`

Yeah I saw that but I have not knowledge about coding and stuff, how do I run `mvn exec:java`
